test/functional/legacy/036_regexp_character_classes_spec.lua

```diff
@@ -270,9 +270,9 @@ describe('character classes in regexp', function()
   end)
   it('is working', function()
     source([[
-      1 s/\%#=0\^\t...//g
-      2 s/\%#=1\^\t...//g
-      3 s/\%#=2\^\t...//g]])
+      1 s/\%#=0\%1l^\t...//g
+      2 s/\%#=1\%2l^\t...//g
+      3 s/\%#=2\%3l^\t...//g]])
     diff(nlines(3, ctrl1..punct1..digits..punct2..punct3..lower..punct4..
```

Comment:
`ctrl1` won't be in the lines because the `:s` commands removed it (along with the first space in punct1).